### PR TITLE
updated video failed example 1

### PR DESCRIPTION
**Pull request by zalmeraj**
Need for Final Call: None

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Review by ajanec01 — APPROVED** [review]

File: _rules/video-alternative-for-visual-c5a4ea.md [hunk 1]
```diff
@@ -155,7 +155,7 @@ A video element that describes some of the text on the same page. The text on th
 
 #### Failed Example 1
 
-A video element with an incorrect audio description.
+This 'video' element has an incorrect audio description.
```
Comment (Jym77) [inline _rules/video-alternative-for-visual-c5a4ea.md:158]:
```suggestion
This `video` element has an incorrect audio description.
```
Text enclosed in backticks is rendered as code, which is what we want here as we refer to the name of the HTML element.
('video' vs `video`)

**Review by Jym77 — CHANGES_REQUESTED** [review]
If you feel confident, feel free to update the other examples description in the same way 👍

**Review by Jym77 — APPROVED** [review]
Github "pro tip":
After making changes requested by a review, you can
1. "dismiss" that review. This happens near the bottom of the page in the drop-down with the "1 change requested" label. => This ensures that other potential reviewers see that these concerns have been addressed; some reviewers are reluctant to review a PR which already has changes requested since it can sometimes feel "double work".
2. "re-request" a review from the same reviewer. This happens near the top of the page, by using the "recycling" arrows next to that reviewer's name. => This ensures that this reviewer gets notified that their concern have been handle and they need to get another look at it.

(no worries if you forget to do it)

**Review by WilcoFiers — APPROVED** [review]

**Comment by WilcoFiers** [comment]
Looks like this is good to go.

**Pull request merged**